### ISSUE TITLE
double clicking in rename input now commits instead of selects word

### ISSUE THREAD

**Issue by yume-chan** [Microsoft/vscode#70276]
```
Version: 1.33.0-insider
Commit: d77d8d561143cc021a82664efc056755c075d070
Date: 2019-03-11T08:47:23.627Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
```

Steps to Reproduce:

1. Press <kbd>F2</kbd> in explorer to enter renaming mode
2. Double click in the input box

Expected Result:

* The word under cursor is selected.

Actual Result:

* Exited renaming mode.

Last known working version:

* 1.30.2

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@yume-chan good catch. Tomorrow's vscode insiders will contain the fix, so let us know if it behaves as expected for you.

**Comment by isidorn** [comment]
Fixed via 900f10077c
